src/pages/about/index.js

```diff
@@ -108,7 +108,7 @@ const About = () => {
   const getProps = () => {
     setIsLoading(true);
     createNewRequest(
-      espHttpURL("command", { cmd: "[ESP420]" }).toString(),
```

Comment:
https://raw.githubusercontent.com/wiki/luc-github/ESP3D/docs/Commands2_1.txt
```
*Get current settings of ESP3D
output is JSON or plain text according parameter
[ESP420]<plain>
```

I am not sure why the `plain` flag was not ported to GRBL_ESP32 but as discussed in discord instead of creating new  command - I am in favor of an output flag for formating the output, it has been the case for several command since a while in ESP3D

Reply:
The <plain> modifier for ESP420 is an okay solution.  Historically, Grbl_Esp32 code received [ESP420]plain but did not look at the "plain" parameter, always returning a plaintext response even if plain was not present.  Thus I did not realize that there was a choice.  Going forward, I can modify FluidNC to depend on plain or its lack.

The situation with ESP800 is not as good, because WebUI 2 issues [ESP800], not [ESP800]plain - so already-deployed firmware is not informed of the different expectation.  But since we are maintaining our own interim version of WebUI 2, we can make a version of WebUI 2 that does issue [ESP800]plain .  That will let to continue to support WebUI 2 while migrating to WebUI 3.

Adding a command-specific parameter with a fixed value is better than using the esp3d version number, because the fixed parameter value is precise.  Version numbers are terrible because they change frequently and already-deployed command handler code has no way to know when new esp3d version number changes the meaning of that command.

The parameter method, while better than an ever-changing version number, is worse than using a different command name.  Existing command handler code that does not currently expect a parameter has no way to determine that a change in expected behavior has occurred.  When a new version of WebUI supplies a new parameter, the old code will just not notice the parameter, and will issue the old, incorrect response.